README.md

```diff
@@ -1,35 +1,82 @@
+[![Build Status](https://app.travis-ci.com/molgenis/vip-inheritance-matcher.svg?branch=main)](https://app.travis-ci.com/molgenis/vip-inheritance-matcher)
 [![Quality Status](https://sonarcloud.io/api/project_badges/measure?project=molgenis_vip-inheritance-matcher&metric=alert_status)](https://sonarcloud.io/dashboard?id=molgenis_vip-inheritance-matcher)
 # Variant Interpretation Pipeline - Inheritance Matcher
 annotates VCF samples with denovo and possible compound flags and matching inheritance modes and genes.
 
-Input VCF file should be annotated with [Genmod models](http://moonso.github.io/genmod/) and the [VIP inheritance VEP plugin](https://github.com/molgenis/vip/tree/master/plugins/vep).
+Input VCF file should contain single ALT alleles per line and be annotated VEP.
+Input should be annotated with [VIP inheritance VEP plugin](https://github.com/molgenis/vip/tree/master/plugins/vep) For full functionality.
 
 ###Added Sample information
 ```
-##FORMAT=<ID=VIC,Number=1,Type=Integer,Description="Inheritance Compound status.">
+##FORMAT=<ID=VI,Number=.,Type=String,Description="An enumeration of possible inheritance modes.">
+##FORMAT=<ID=VIC,Number=1,Type=String,Description="List of possible compounds.">
 ##FORMAT=<ID=VID,Number=1,Type=Integer,Description="Inheritance Denovo status.">
 ##FORMAT=<ID=VIG,Number=.,Type=String,Description="Genes with an inheritance match.">
-##FORMAT=<ID=VIM,Number=1,Type=String,Description="Inheritance Match status.">
-##FORMAT=<ID=VI,Number=.,Type=String,Description="An enumeration of possible inheritance modes.">
+##FORMAT=<ID=VIM,Number=1,Type=Integer,Description="Inheritance Match status.">
+##FORMAT=<ID=VIS,Number=.,Type=String,Description="An enumeration of possible sub inheritance modes like e.g. compound, non penetrance.">
 ```
 
 ###Usage
 ```
 usage: java -jar vcf-inheritance-matcher.jar -i <arg> [-o <arg>] [-pd
-       <arg>] [-pb <arg>] [-f] [-d]
- -i,--input <arg>       Input VCF file (.vcf or .vcf.gz).
- -o,--output <arg>      Output VCF file (.vcf or .vcf.gz).
- -pd,--pedigree <arg>   Comma-separated list of pedigree files (.ped).
- -pb,--probands <arg>   Comma-separated list of proband sample
-                        identifiers.
- -f,--force             Override the output file if it already exists.
- -d,--debug             Enable debug mode (additional logging).
-
-usage: java -jar vcf-inheritance-matcher.jar -v
- -v,--version   Print version.
+       <arg>] [-pb <arg>] [-np <arg>] [-c] [-f] [-d]
+ -i,--input <arg>            Input VCF file (.vcf or .vcf.gz).
+ -o,--output <arg>           Output VCF file (.vcf or .vcf.gz).
+ -pd,--pedigree <arg>        Comma-separated list of pedigree files
+                             (.ped).
+ -pb,--probands <arg>        Comma-separated list of proband sample
+                             identifiers.
+ -np,--nonpenetrance <arg>   File containing a list of non penetrance
```

Comment:
terminology: shouldn't this be reduced penetrance instead of non penetrance?

Reply:
depends on who you ask I guess, this is the term our department uses I think